### PR TITLE
fix - cat a mod1 test report allow second attempt to be entered with a 0 on first attempt

### PR DESCRIPTION
**Pull request by appleabs**
## Description
fix - cat a mod1 test report allow second attempt to be entered with a 0 on first attempt
## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

**Review by ammarhaiderbjss — COMMENTED** [review]
Would be nice to have this as a method on the component that we can unit test given there's a bit of logic going on

**Review by rossfauldsbjss — APPROVED** [review]
looks good

File: src/pages/test-report/cat-a-mod1/components/speed-check/speed-check.ts [hunk 1]
```diff
@@ -305,4 +305,8 @@ export class SpeedCheckComponent {
     }
   }
 
+  firstAttemptValid(): boolean {
+    return this.firstAttempt >= 0 && this.firstAttempt !== null;
```
Comment (ammarhaiderbjss) [inline src/pages/test-report/cat-a-mod1/components/speed-check/speed-check.ts]: Perhaps more optimal to have the null check first.

**Review by ammarhaiderbjss — APPROVED** [review]
Lovely 😍 one minor comment

**Pull request merged**